src/legacy/core_plugins/input_control_vis/public/components/vis/_vis.scss

```diff
@@ -21,6 +22,11 @@
   }
 }
 
+.icvContent {
+  box-sizing: border-box;
```

Comment:
You don't need to set this here because it's already added in the EUI reset.

![image](https://user-images.githubusercontent.com/549577/52579722-bf85d500-2df4-11e9-9c7c-5b05f8c874c5.png)